### PR TITLE
[SDK-3710] Readme Redesign

### PR DESCRIPTION
**Pull request by stevehobbsdev**
- [x] Has header image

image

### CODE REVIEW

File: EXAMPLES.md [hunk 1]
```diff
@@ -0,0 +1,193 @@
+## Build a URL to the Universal Login Page
```
Comment (frederikprijck) [inline EXAMPLES.md:1]:
```suggestion
# Examples using ruby-auth0

## Build a URL to Universal Login Page
```

File: README.md [hunk 2]
```diff
@@ -1,116 +1,74 @@
...
 
 [![CircleCI](https://img.shields.io/circleci/project/github/auth0/ruby-auth0/master.svg)](https://circleci.com/gh/auth0/ruby-auth0)
 [![Gem Version](https://badge.fury.io/rb/auth0.svg)](http://badge.fury.io/rb/auth0)
 [![codecov](https://codecov.io/gh/auth0/ruby-auth0/branch/master/graph/badge.svg)](https://codecov.io/gh/auth0/ruby-auth0)
 [![Yard Docs](http://img.shields.io/badge/yard-docs-blue.svg)](http://www.rubydoc.info/github/auth0/ruby-auth0/master/frames)
 [![MIT licensed](https://img.shields.io/dub/l/vibe-d.svg?style=flat)](https://github.com/auth0/ruby-auth0/blob/master/LICENSE)
-[![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fauth0%2Fruby-auth0.svg?type=shield)](https://app.fossa.com/projects/git%2Bgithub.com%2Fauth0%2Fruby-auth0?ref=badge_shield)
 
-Ruby API client for the [Auth0](https://auth0.com) platform.
+:books: [Documentation](#documentation) - :rocket: [Getting Started](#getting-started) - :computer: [API Reference](https://www.rubydoc.info/gems/auth0) - :speech_balloon: [Feedback](#feedback)
```
Comment (frederikprijck) [inline README.md:11]: As I had some troubles with this for python, does rubydoc support these emoji's ? It looks like it uses the readme as the package's homepage: https://www.rubydoc.info/gems/auth0

I solved it in python as seen here: https://github.com/auth0/auth0-python/pull/448

File: README.md [hunk 3]
```diff
@@ -123,194 +81,50 @@ In addition to the Management API, this SDK also provides access to [Authenticat
...
+Please do not report security vulnerabilities on the public GitHub issue tracker. The [Responsible Disclosure Program](https://auth0.com/whitehat) details the procedure for disclosing security issues.
 
+## What is Auth0?
 
-[![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fauth0%2Fruby-auth0.svg?type=large)](https://app.fossa.com/projects/git%2Bgithub.com%2Fauth0%2Fruby-auth0?ref=badge_large)
+<p align="center">
+  <picture>
+    <source media="(prefers-color-scheme: dark)" srcset="./auth0_dark_mode.png" width="150">
+    <source media="(prefers-color-scheme: light)" srcset="./auth0_light_mode.png" width="150">
+    <img alt="Auth0 Logo" src="./auth0_light_mode.png" width="150">
```
Comment (frederikprijck) [inline README.md:122]: Will this work on rubydocs, should we configure the images as assets to be sent to the api docs somewhere?

**Review by frederikprijck — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -123,194 +81,50 @@ In addition to the Management API, this SDK also provides access to [Authenticat
...
 
-[Auth0](https://auth0.com)
+### Vulnerability Reporting
 
-## License
-
-This project is licensed under the MIT license. See the [LICENSE](LICENSE) file for more info.
+Please do not report security vulnerabilities on the public GitHub issue tracker. The [Responsible Disclosure Program](https://auth0.com/whitehat) details the procedure for disclosing security issues.
 
+## What is Auth0?
```
Comment (frederikprijck) [inline README.md:116]:
```suggestion
---
```

**Review by frederikprijck — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (stevehobbsdev) [reply README.md:122]: It's a great idea, see: https://auth0.slack.com/archives/C0414QJGXG9/p1666171204015449?thread_ts=1666170245.671199&cid=C0414QJGXG9

File: README.md (hunk 2 above)
Comment (stevehobbsdev) [reply README.md:11]: It's a good question, I don't know. I will use your method from Python anyway as a precaution 👍🏻

File: README.md (hunk 3 above)
Comment (stevehobbsdev) [reply README.md]: I've used logos from the CDN now.

**Review by frederikprijck — DISMISSED** [review]

File: README.md [hunk 5]
```diff
@@ -123,194 +83,50 @@ In addition to the Management API, this SDK also provides access to [Authenticat
...
-If you find a bug or have a feature request, please report them in this repository's [Issues tab](https://github.com/auth0/ruby-auth0/issues). Please do not report security vulnerabilities on the public GitHub issue tracker. The [Responsible Disclosure Program](https://auth0.com/whitehat) details the procedure for disclosing security issues.
+- [Auth0's general contribution guidelines](https://github.com/auth0/open-source-template/blob/master/GENERAL-CONTRIBUTING.md)
+- [Auth0's code of conduct guidelines](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md)
+- [The contribution guide](https://github.com/auth0/open-source-template/blob/master/GENERAL-CONTRIBUTING.md)
 
-## Author
+### Raise an issue
 
-[Auth0](https://auth0.com)
+To provide feedback or report a bug, please [raise an issue on our issue tracker](https://github.com/auth0/auth0-flutter/issues).
```
Comment (frederikprijck) [inline README.md:112]: Let's not try and solve ruby issues in the flutter SDK 😆 

```suggestion
To provide feedback or report a bug, please [raise an issue on our issue tracker](https://github.com/auth0/ruby-auth0/issues).
```

File: README.md [hunk 6]
```diff
@@ -123,194 +83,50 @@ In addition to the Management API, this SDK also provides access to [Authenticat
...
-2. Use Google, GitHub or Microsoft Account to login.
+### Contributing
 
-## Issue Reporting
+We appreciate feedback and contribution to this repo! Before you get started, please see the following:
 
-If you find a bug or have a feature request, please report them in this repository's [Issues tab](https://github.com/auth0/ruby-auth0/issues). Please do not report security vulnerabilities on the public GitHub issue tracker. The [Responsible Disclosure Program](https://auth0.com/whitehat) details the procedure for disclosing security issues.
+- [Auth0's general contribution guidelines](https://github.com/auth0/open-source-template/blob/master/GENERAL-CONTRIBUTING.md)
+- [Auth0's code of conduct guidelines](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md)
+- [The contribution guide](https://github.com/auth0/open-source-template/blob/master/GENERAL-CONTRIBUTING.md)
```
Comment (frederikprijck) [inline README.md:108]:
```suggestion
```

File: README.md [hunk 7]
```diff
@@ -1,116 +1,76 @@
...
 
-Ruby API client for the [Auth0](https://auth0.com) platform.
+<div>
+📚 <a href="#documentation">Documentation</a> - 🚀 <a href="#getting-started">Getting started</a> - 💻 <a href="#api-reference">API reference</a> - 💬 <a href="#feedback">Feedback</a>
+</div>
+
+## Documentation
+
+- [API documentation](https://www.rubydoc.info/gems/auth0) - documentation auto-generated from the code comments that explains all the available features
+- [Examples](EXAMPLES.md) - examples that demonstrate the different ways in which this SDK can be used
```
Comment (frederikprijck) [inline README.md:18]:
```suggestion
- [Examples](https://github.com/auth0/ruby-auth0/blob/master/EXAMPLES.md) - examples that demonstrate the different ways in which this SDK can be used
```

File: README.md [hunk 8]
```diff
@@ -1,116 +1,76 @@
...
+- [FAQ](FAQ.md) - frequently asked questions about the SDK
```
Comment (frederikprijck) [inline README.md:19]:
```suggestion
[FAQ](https://github.com/auth0/ruby-auth0/blob/master/FAQ.md) - frequently asked questions about the SDK
```

File: README.md [hunk 9]
```diff
@@ -123,194 +83,50 @@ In addition to the Management API, this SDK also provides access to [Authenticat
...
+    <source media="(prefers-color-scheme: dark)" srcset="https://cdn.auth0.com/website/sdks/logos/auth0_dark_mode.png" width="150">
+    <source media="(prefers-color-scheme: light)" srcset="https://cdn.auth0.com/website/sdks/logos/auth0_light_mode.png" width="150">
+    <img alt="Auth0 Logo" src="https://cdn.auth0.com/website/sdks/logos/auth0_light_mode.png" width="150">
+  </picture>
+</p>
+<p align="center">
+  Auth0 is an easy to implement, adaptable authentication and authorization platform. To learn more checkout <a href="https://auth0.com/why-auth0">Why Auth0?</a>
+</p>
+<p align="center">
+  This project is licensed under the MIT license. See the <a href="./LICENSE"> LICENSE</a> file for more info.
```
Comment (frederikprijck) [inline README.md:131]:
```suggestion
  This project is licensed under the MIT license. See the <a href="https://github.com/auth0/ruby-auth0/blob/master/LICENSE"> LICENSE</a> file for more info.
```

**Review by frederikprijck — CHANGES_REQUESTED** [review]
Looks like rubygem doesn't like relative links (e.g. currently, the link to the license file is giving a 404 on rubygems)

**Comment by stevehobbsdev** [comment]
Doh! Will fix shortly 👍

File: README.md [hunk 10]
```diff
@@ -123,194 +83,49 @@ In addition to the Management API, this SDK also provides access to [Authenticat
...
-[Organizations](https://auth0.com/docs/organizations) is a set of features that provide better support for developers who build and maintain SaaS and Business-to-Business (B2B) applications.
-
-Note that Organizations is currently only available to customers on our Enterprise and Startup subscription plans.
+## Management API Client
 
-#### Logging in with an Organization
+This SDK provides access to the [Management API](https://auth0.com/docs/api/management/v2) via modules that help create clear and accurate calls. Most of the interaction is done through the `Auth0Client` class, instantiated with the required credentials.
 
-Configure the Authentication API client and pass your Organization ID to the authorize url:
+For an example of using the management API client to read of users, see the [examples document](EXAMPLES.md).
```
Comment (frederikprijck) [inline README.md:90]:
```suggestion
For an example of using the management API client to read of users, see the [examples document](https://github.com/auth0/ruby-auth0/blob/master/EXAMPLES.md).
```

**Review by frederikprijck — CHANGES_REQUESTED** [review]

**Review by frederikprijck — APPROVED** [review]

**Pull request merged**